apps/pie-docs/src/accessibility/accessibility-overview.md

```diff
@@ -0,0 +1,82 @@
+---
+eleventyNavigation:
+    key: Accessibility overview
+    parent: Accessibility
+    order: 1
+title: Accessibility overview
+description: In this section you’ll find useful information about accessibility and why it is important at JustEatTakeaway.com
+---
+
+## Introduction
+
+Welcome to the **Accessibility** section of PIE design system’s documentation site. Our goal is to make our platforms better for everyone by creating inclusive and user-friendly products. We are committed to accessibility based on the principles of universal design and inclusivity.
+
+{% contentPageImage {
+    src:"../../../assets/img/accessibility/accessibility-overview/accessibility-introduction.svg",
+    mobileSrc:"../../../assets/img/accessibility/accessibility-overview/accessibility-introduction_narrow.svg",
+    caption: "Inclusive design is better design for everyone, no user should be excluded because of disability."
+} %}
+
+---
+
+## Why is accessibility important?
+
+Just as a good recipe caters to the diverse tastes and dietary needs of all diners, a design system that incorporates accessibility ensures that all users, **regardless of their abilities or disabilities**, can access and use digital products and services.
+
+Incorporating accessibility into a design system can lead to a **better user experience, increased engagement**, and **increased customer loyalty**.
+
+Finally, it’s important to highlight that designing with accessibility in mind can help organisations comply with accessibility standards and regulations, avoiding **legal and financial consequences**.
+
+---
+
+## How we build inclusive experiences
+
+Using our components is a way to improve accessibility and consistency when building products for JustEatTakeaway.com.
+
+The component library in this documentation site includes code that we can use across applications, and this code includes **accessible markup**. Since the code exists in a single component that gets reused, it's easier to update and fix any bugs.
+
+The build-it-once, use-it-everywhere model means the **accessibility knowledge** of the designers and developers who build these components is available to all of JET and our partners.
+
+Many accessibility features come free in the components, but it's important to make sure that components are integrated in a way that doesn't create **unforeseen accessibility barriers**. Depending on how components are used, there may be more design and implementation considerations.
+
+---
+
+## Assistive technology support
+
+Our components are tested for accessibility with automated and manual techniques. Users should expect to be able to access features built with our components using **modern assistive technologies**, including native and **third-party tools** like screen readers, speech recognition programs, supports for low vision and colour blindness, alternative keyboards, switch devices, and tools for readability.
```

Comment:
Do we want to mention any of the particular programs and screenreaders we should test on?

Reply:
@jamieomaguire We have a 'Testing for accessibility' section in which we give more details. I can link to that page so we don't overwhelm people in this page already.

I would suggest having a look at that page yourself and adding/removing/changing any bits, if possible, thanks!

Reply:
@FayeCarter let's add a link to the Testing for accessibility page in this paragraph, I'll add it to designs